plugins/modules/vmware_object_custom_attributes_info.py

```diff
@@ -43,7 +43,11 @@
     type: str
     aliases:
       - name
-    required: True
+  moid:
+    description:
+      - Managed Object ID of the instance to get if known, this is a unique identifier only within a single vCenter instance.
+      - This is required if C(object_name) is not supplied.
+    type: str
```

Comment:
```suggestion
    type: str
    version_added: '1.11.0'
```

Reply:
Thanks, @Akasurde for your point out!

This module will be released as version 1.11.0.  
https://github.com/ansible-collections/community.vmware/blob/b35b381181ed393a725a790b3bc64640469bed60/plugins/modules/vmware_object_custom_attributes_info.py#L23

So I didn't add the version_added to the new option.  
But, is it better should add the version_added?

What do you think?

Reply:
Oh, I didn't realize that. No need to add this then.